src/components/UniversalNavbarExpanded/DropdownNav/DropdownChildren.module.scss

```diff
@@ -0,0 +1,94 @@
+@import '../../Media/Media.scss';
+@import '../../Variables.scss';
+
+$laptop-range-medium-end: 1050px;
+
+.dropdownNavChildren {
+  display: none;
+  position: absolute;
+  top: var(--Space-64);
+  left: 0;
+  width: 100%;
+  background: var(--White);
+  border-bottom: $navbar-border;
+  min-height: 220px;
+}
+
+.dropdownNavChild {
+  margin-bottom: var(--Space-16);
+  a {
+    text-decoration: none;
+    &:hover {
+      text-decoration: none;
+      opacity: 0.78;
```

Comment:
For Nora, we've created `src/nora/components/Attributes.css` for storing these sorts of things. Maybe we should create a top-level `Attributes.css` in the spirit of what we already do for spacing and colors? Also, I presume .78 just somehow looked the best? I suppose it's fine, but especially will be better if it's a shared value I think. Pretty sure our `opacity` is all over the map at the moment.

Reply:
Okay, I'll try and follow that pattern. 

Yes 0.78 was just the spec from Figma.

Reply:
dd7d058